telegram/ext/dispatcher.py

```diff
@@ -365,5 +365,13 @@ def dispatch_error(self, update, error):
             error (telegram.TelegramError): The Telegram error that was raised.
         """
 
-        for callback in self.error_handlers:
-            callback(self.bot, update, error)
+        if self.error_handlers:
+            for callback in self.error_handlers:
+                callback(self.bot, update, error)
+
+        else:
+            try:
+                raise error
+
+            except:
+                self.logger.exception('No error handlers are registered, logging exception...')
```

Comment:
Maybe it'll be better to use `self.logger.exception('...', exc_info=error)` instead of this try-except clause?